interop-cats/shared/src/main/scala/zio/interop/catszmanaged.scala

```diff
@@ -124,7 +124,7 @@ private class CatsZManagedMonad[R, E] extends Monad[ZManaged[R, E, ?]] {
   override def flatMap[A, B](fa: ZManaged[R, E, A])(f: A => ZManaged[R, E, B]): ZManaged[R, E, B] = fa.flatMap(f)
 
   override def tailRecM[A, B](a: A)(f: A => ZManaged[R, E, Either[A, B]]): ZManaged[R, E, B] =
-    ZManaged.succeedLazy(f(a)).flatMap(identity).flatMap {
+    ZManaged.effectTotal(f(a)).flatMap(identity).flatMap {
```

Comment:
The tests should now pass with `ZManaged.suspend` instead of this construct. Could you please check?

Reply:
Ok I’ll try